### PR TITLE
chore(fake-driver): annihilate loathsome mixin classes

### PR DESCRIPTION
**Pull request by boneskull**
I did here what I did to `appium-xcuitest-driver`, which was to assign the command methods to the class as fields.  Easier this way.

Also, it's more difficult to parse for automatic command docs using the old way.

### CODE REVIEW

**Comment by boneskull** [comment]
Current dependencies on/for this PR:
* master
  * **PR #18058** <a href="https://app.graphite.dev/github/pr/appium/appium/18058" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/appium/appium/18058?utm_source=stack-comment).

File: packages/fake-driver/lib/commands/alert.js [hunk 1]
```diff
@@ -1,82 +1,73 @@
 import {errors} from 'appium/driver';
 
-/**
- * @template {Class<import('../types').IContextsCommands & import('@appium/types').IFindCommands>} T
- * @param {T} Base
- * @returns {Class<AlertCommands>}
- */
-export function AlertMixin(Base) {
+export default {
```
Comment (jlipps) [inline packages/fake-driver/lib/commands/alert.js:3]: why `export default` instead of `export async function` for each one?

**Review by jlipps — APPROVED** [review]

**Comment by mykola-mokhnach** [comment]
LGTM, but tests seem to be unhappy

File: packages/fake-driver/lib/commands/alert.js (hunk 1 above)
Comment (boneskull) [reply packages/fake-driver/lib/commands/alert.js:3]: six and 1/2 dozen I guess

Comment (boneskull) [reply packages/fake-driver/lib/commands/alert.js:3]: @jlipps I could go either way here. given these aren't intended to be imported by anything other than `commands/index.js`, if you have a preference I can change it.

Comment (boneskull) [reply packages/fake-driver/lib/commands/alert.js:3]: I'll change it later if you wish

**Pull request merged**